include/seqan3/alignment/configuration/align_config_gap_cost_affine.hpp

```diff
@@ -0,0 +1,110 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Provides seqan3::align_config::gap_cost_affine.
+ * \author Rene Rahn <rene.rahn AT fu-berlin.de>
+ * \author Jörg Winkler <j.winkler AT fu-berlin.de>
+ * \author Hannes Hauswedell <hannes.hauswedell AT fu-berlin.de>
+ * \author Wiep van der Toorn <w.vandertoorn AT fu-berlin.de>
+ */
+
+#pragma once
+
+#include <seqan3/alignment/configuration/detail.hpp>
+#include <seqan3/core/algorithm/pipeable_config_element.hpp>
+#include <seqan3/core/detail/strong_type.hpp>
+
+namespace seqan3::align_cfg
+{
+// ------------------------------------------------------------------
+// seqan3::gap_open_score
+// ------------------------------------------------------------------
+
+/*!\brief A strong type of underlying type int32_t` that represents a score (usually negative) that
```

Comment:
I'm not sure, but I think you missed a ` here :)
```suggestion
/*!\brief A strong type of underlying type `int32_t` that represents a score (usually negative) that
```